### ISSUE TITLE
notebook 5-nn-imdb, issue: Can't call numpy() on Variable that requires grad. Use var.detach().numpy() instead.

### ISSUE THREAD

**Issue by RaffaToSpace** [fastai/course-nlp#47]
It happens at:

np.allclose(enc.weight[vocab.stoi["30-something"], :], 
            enc.weight[vocab.stoi["linklater"], :])

numpy version 1.19.1
python 3.6.11 (can't upgrade due to compatibility with tensorflow version for CUDA capability 3.0)
pythorch 1.1.0
fastAI 1.0.61

Thanks
